core/print_api.php

```diff
@@ -1984,3 +1984,21 @@ function print_max_filesize( $p_size, $p_divider = 1000, $p_unit = 'kb' ) {
 		. get_filesize_info( $p_size / $p_divider, lang_get( $p_unit ) );
 	echo '</span>';
 }
+
+/**
+ * Print required js for datetimepicker
+ * @return void
+ * @access public
+ */
+function print_datetimepicker_js() {
```

Comment:
Does this belong better in html_api.php rather than print_api.php?  html is where we add the css/js/etc and print is where we print data and other info.